Validation/RecoTrack/plugins/MultiTrackValidator.cc

```diff
@@ -637,6 +677,13 @@ void MultiTrackValidator::dqmAnalyze(const edm::Event& event, const edm::EventSe
   declareDynArray(float, tPCeff.size(), dR_tPCeff);
   size_t n_selTP_dr = tpDR(tPCeff, selected_tPCeff, dR_tPCeff);
 
+  //calculate dR_jet for TPs
+  Handle<edm::View<reco::Candidate> > cores;
+  event.getByToken(cores_, cores);
+  const auto & coresVector = cores.product();
```

Comment:
```suggestion
  const auto & coresVector = *cores;
```
to have the type of `coresVector` not to be a pointer.